13/umbraco-deploy/deploy-settings.md

```diff
@@ -109,28 +109,15 @@ This setting allows you to manage how relations are deployed between environment
 * `Weak` - This causes the relation to be deployed if both content items are found on the target environment.
 * `Strong` - This requires the content item that is related is set as a dependency, so if anything is added as a relation it would also add it as a dependency.
 
-```json
-"RelationTypes": [
-    {
-        "Alias": "relateParentDocumentOnDelete",
-        "Mode": "Weak",
-    },
-    {
-        "Alias": "relateShopItemOnCreate",
-        "Mode": "Exclude",
-    }
-],
```

Comment:
The format of this setting has changed in v13 (see breaking changes) and is already shown below...

Reply:
should this be removed only for v13 or for all versions?

Reply:
Only for v13, as that uses the format that's already shown below.